### PR TITLE
ci: check license header tool

### PR DESCRIPTION
**Pull request by tisonkun**
Use hawkeye to check license headers. Apply with the standard Apache License 2.0 header.

You can review in commits and skip the globally apply changes one.

### CODE REVIEW

**Comment by tisonkun** [comment]
cc @ShiKaiWi @ZuLiangWang

**Review by tanruixiang — APPROVED** [review]
LGTM. It looks like we have a problem with the inception year in the [Ceresdb](https://github.com/CeresDB/ceresdb/blob/main/licenserc.toml) license header and should change it to 2022. 🤔

File: Makefile [hunk 1]
```diff
@@ -1,7 +1,3 @@
-# Refers to https://github.com/tikv/pd/blob/master/Makefile
-
-default: build
-
```
Comment (tanruixiang) [inline Makefile:4]: What I'm not sure about is, do we need to keep the link?

Comment (tisonkun) [reply Makefile:4]: Maybe. But I don't find any similarities this Makefile with pd's.

Comment (tanruixiang) [reply Makefile:4]: @chunshao90 Do you remember why we added this link before? 🤣

Comment (ShiKaiWi) [reply Makefile:4]: I guess there are some similarities between them: https://github.com/tikv/pd/blob/master/Makefile#L143

Comment (tisonkun) [reply Makefile:4]: They are quite different and if it's not clearly driven from another file, we should try to reduce the IP burden.

**Review by ShiKaiWi — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (tanruixiang) [reply Makefile:4]: > They are quite different and if it's not clearly derived from another file, we should try to reduce the IP burden.

Makes sense to me.

**Pull request merged**